yb-voyager/cmd/common_test.go

```diff
@@ -0,0 +1,396 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/migassessment"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+)
+
+func TestAssessmentReportStructs(t *testing.T) {
+	excpectedDBObject := struct {
+		ObjectType   string `json:"ObjectType"`
+		TotalCount   int    `json:"TotalCount"`
+		InvalidCount int    `json:"InvalidCount"`
+		ObjectNames  string `json:"ObjectNames"`
+		Details      string `json:"Details,omitempty"`
+	}{}
+
+	t.Run("Check DBObject structure", func(t *testing.T) {
```

Comment:
nit: `Check DBObject structure` --> `Validate DBObject Struct Definition`
similarly for others.

Reply:
Done